### DISCUSSION TITLE
Unable to run on windows container

### DISCUSSION THREAD

**Discussion by iHarshalAgarwal** [dotnet/dotnet-docker#2480] [Q&A] [answered]
Hi, I was able to run successfully in a Linux container. But not in Windows container.

Any suggestions?

**Comment by MichaelSimons** [comment, marked as answer]
@iHarshalAgarwal, Can you include the steps you have tried?

**Reply by iHarshalAgarwal** [reply]
@MichaelSimons, I followed below steps:
Switch to windows container 
Run command at _dotnet-docker/samples/aspnetapp_ - docker build -t aspnetapp .

I see the below output
> Sending build context to Docker daemon  9.107MB
> Step 1/12 : FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
> 5.0: Pulling from dotnet/sdk
> no matching manifest for windows/amd64 10.0.17134 in the manifest list entries

Thanks!

**Reply by mthalman** [reply]
You're running this from a machine with Windows, version 1803 installed.  Since the tag you are targeting is a multi-platform tag, Docker attempts to resolve that to an image that matches the Windows version you have.  We do not provide images for Windows Server, version 1803 because it is [out of support](https://docs.microsoft.com/lifecycle/announcements/windows-server-1803-end-of-servicing).  The oldest version of Windows that we provide images for is version 1809, so you'll need to use a machine that has that version or higher installed.

**Reply by iHarshalAgarwal** [reply]
thanks @mthalman . I am on organization system, so I cannot upgrade it myself.

I have another issue, if you/anyone reading this could guide, I am unable to connect to internet network or any of my internal networks when on windows container. However, I can connect the same from Linux containers.
Any suggestions on that?
